File: Raven.Client.Java/src/main/java/net/ravendb/abstractions/indexing/MergeSuggestions.java
package net.ravendb.abstractions.indexing;

import java.util.ArrayList;
import java.util.List;


public class MergeSuggestions {
  private List<String> canMerge = new ArrayList<>();
  private IndexDefinition mergedIndex = new IndexDefinition();

  public List<String> getCanMerge() {
    return canMerge;
  }

  public void setCanMerge(List<String> canMerge) {
    this.canMerge = canMerge;
  }

  public IndexDefinition getMergedIndex() {
    return mergedIndex;
  }

  public void setMergedIndex(IndexDefinition mergedIndex) {
    this.mergedIndex = mergedIndex;
  }

}

File: src/Raven.NewClient/Indexing/MergeSuggestions.cs
using System;
using System.Collections.Generic;

using Raven.NewClient.Client.Indexing;

namespace Raven.Abstractions.Indexing
{
    public class MergeSuggestions
    {
        public IndexDefinition MergedIndex = new IndexDefinition();  //propose for new index with all it's properties

        // start MergedIndex != null
        public List<string> CanMerge = new List<string>();  // index names

        public string Collection = String.Empty; // the collection that is being merged
        // end MergedIndex != null

        // start MergedIndex == null
        public List<string> CanDelete = new List<string>();  // index names

        public string SurpassingIndex = string.Empty;
        // end MergedIndex == null
    }
}
